acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/publickeys/KeyIdentifiersAcceptanceTestBase.java

```diff
@@ -153,6 +154,8 @@ private void createSecpKey(final String privateKeyHexString) {
   protected void initAndStartSigner(final String mode) {
     final SignerConfigurationBuilder builder = new SignerConfigurationBuilder();
     builder.withKeyStoreDirectory(testDirectory).withMode(mode);
+    if (mode.equals("eth1"))
+      builder.withChainIdProvider(new ConfigurationChainId(DEFAULT_CHAIN_ID));
```

Comment:
Did we discuss this already on another PR? Maybe it's OK to always set the chainId? Not sure.

Reply:
removed, think it crawled back somehow when I reverted my previous PR